Dockerfile

```diff
@@ -22,9 +22,16 @@ RUN apt-get update \
      wget \
   && wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
   && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
+  && curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
```

Comment:
Could we remove wget and use curl everywhere?

Reply:
i'll do this in a different pr. decided not to touch the `Dockerfile` in this one